tools/apigee-x-trial-provision/README.md

```diff
@@ -127,3 +127,180 @@ the `CERTIFICATES` environment variable:
 |`generated`|Auto-Generated Self-Signed Certs|$RUNTIME_HOST_ALIAS or else $ORG-eval.apigee.net|
 |`supplied`|User-supplied in `RUNTIME_SSL_CERT` and `RUNTIME_SSL_KEY`|$RUNTIME_HOST_ALIAS or else $ORG-eval.apigee.net|
 <!-- markdownlint-enable MD013 -->
+
+## Apigee X instance with Custom Networking Provisioning
+<!-- markdownlint-disable MD013 -->
+
+## VPC Network Planning
+
+There is always a step to plan your network with regards to CIDR allocations.
+
+GCP/GKE
+CIDR: 10.0.0.0/14 CIDR IP Range: 10.0.0.0 - 10.3.255.255
+
+CIDR: 10.0.0.0/16 CIDR IP Range: 10.0.0.0 - 10.0.255.255
+
+NOTE: GCP recommended naming convention: <https://cloud.google.com/solutions/best-practices-vpc-design>
+
+### Bastion VM in a default network
+
+As Apigee X instance provisioning is a long-running operation, we would recommend to use your working PC terminal or to provision a bastion VM. Bastion VM is also useful for troubleshooting.
+
+1. In the GCP Console, select Compute Engine/VM instances hamburger menu item.
+
+1. Press Create Instance button
+
+1. Accept defaults and press Create button at the bottom of the page
+
+1. SSH into the Bastion VM instance-1. Press the SSH button.
+
+1. Login into your account to have necessary permissions.
+
+    ```sh
+    gcloud auth login --quiet
+    ```
+
+### Custom Network and Subnet Creation
+
+1. Define PROJECT variable by the ID of your project
+
+    ```sh
+    export PROJECT=<project-id>
+    ```
+
+1. Define environment variables that describe network topology
+
+    ```sh
+    export NETWORK=exco-vpc
+    export SUBNET=exco-vpc-dev-subnet
+
+    export REGION=us-central1
+    export ZONE=us-central1
+    export AX_REGION=us-central1
+
+    export NETWORK_CIDR=10.0.0.0/14
+    export SUBNET_CIDR=10.0.0.0/16
+    ```
+
+1. GCP: Create a VPC Network
+
+    ```sh
+    gcloud compute networks create $NETWORK \
+        --subnet-mode=custom \
+        --bgp-routing-mode=regional \
```

Comment:
remove defaults (gbp-routing-mode and mtu)

Reply:
Fixed.